components/gitpod-cli/cmd/rebuild.go

```diff
@@ -268,9 +268,10 @@ func runRebuild(ctx context.Context, supervisorClient *supervisor.SupervisorClie
 		"-p", "25003:23003", // debug workspace proxy
 
 		// volumes
-		"-v", "/workspace:/workspace",
+		"-v", checkoutLocation+":"+checkoutLocation,
 		"-v", "/.supervisor:/.supervisor",
-		"-v", "/var/run/docker.sock:/var/run/docker.sock",
+		"-v", "/usr/bin/docker-up:/usr/bin/docker-up",
```

Comment:
@iQQBot not sure whether we can do anything to make docker run in the inner container?